packages/tupaia-web-server/src/app/createApp.ts

```diff
@@ -81,15 +81,10 @@ export function createApp() {
     )
     // TODO: Stop using get for logout, then delete this
     .get<TempLogoutRequest>('logout', handleWith(TempLogoutRoute))
-    .build();
 
-  useForwardUnhandledRequests(
-    app,
-    WEB_CONFIG_API_URL,
-    '',
-    attachSessionIfAvailable,
```

Comment:
I assume we can remove `attachSessionIfAvailable` because we have already used it in line 54 above?